docs/dev/debugger.rst

```diff
@@ -148,6 +148,22 @@ How to use Debugger?
     m.run()
     tvm_out = m.get_output(0, tvm.nd.empty(out_shape, dtype)).numpy()
 
+4. If network previously was exported to external libray using ``lib.export_library("network.so")``
+     like shared object file/dynamic linked library, the initialization
+     of debug runtime will be slightly different
+
+::
+    lib = tvm.runtime.load_module("network.so")
+    m = graph_executor.GraphModuleDebug(lib["debug_create"]("default", dev),
+                                         [dev], lib["get_json"](), dump_root="/tmp/tvmdbg")
```

Comment:
Does `graph_executor.create` not work here?

Reply:
Could you please share how to pass lib to the second param of graph_executor.create? If I pass lib been created through tvm.runtime.load_module directly, it does not work.

Reply:
`graph_executor.create(lib["get_json"](), lib, dev, dump_root="/tmp/tvmdbg")` seems to work for me locally. Maybe I am testing it wrong though?

Reply:
Hm, interesting. I tried the same code and it did not work, but when I tried your code snippet, everything started to work well. Will update the doc. I might not be required, but still there is a difference how to get the json between just compiled lib and loaded from the external dynamic library -`lib["get_json"]()`